graphql/src/main/java/com/github/twitch4j/graphql/TwitchGraphQL.java

```diff
@@ -187,6 +187,10 @@ public CommandUpdateClip updateClip(OAuth2Credential auth, String slug, String n
         return new CommandUpdateClip(getApolloClient(auth), slug, newTitle);
     }
 
+    public CommandFetchChannelPointRewards fetchChannelPointRewards(OAuth2Credential auth, String channelLogin) {
```

Comment:
should we add a comment here referencing the real operation name `communityPointsSettings `?

Reply:
update the class name to reflect the real operation name; better now?